ansible/config_sonic_basedon_testbed.yml

```diff
@@ -174,51 +162,27 @@
         state: directory
         mode: '0755'
       become: true
-
-    - name: Create telemetry server private key
-      openssl_privatekey:
-        path: "{{ server_key }}"
-        size: 2048
-        mode: '0755'
-      become: true
-
-    - name: create telemetry server csr
-      openssl_csr:
-        path: "{{ telemetry_certs['server_csr'] }}"
-        privatekey_path: "{{ server_key }}"
-      become: true
-
-    - name: Generate a Self Signed OpenSSL telemetry server certificate
-      openssl_certificate:
-        path: "{{ server_cer }}"
-        privatekey_path: "{{ server_key }}"
-        csr_path: "{{ server_csr }}"
-        subject:
-          commonName: ndastreamingservertest
-        provider: selfsigned
-      become: true
-
-    - name: Create telemetry dsmsroot private key
-      openssl_privatekey:
-        path: "{{ dsmsroot_key }}"
-        size: 2048
-        mode: '0755'
-      become: true
-
-    - name: create telemetry dsmsroot csr
-      openssl_csr:
-        path: "{{ dsmsroot_csr }}"
-        privatekey_path: "{{ dsmsroot_key }}"
+    
+    - name: Generate server cert using openssl. This need to be copied and renamed as dsmsroot.cer on ptfdocker
+      command: openssl req \
+          -x509 \
+          -sha256 \
+          -nodes \
+          -newkey rsa:2048 \
+          -keyout "{{ server_key }}"
+          -subj "/CN=ndastreamingservertest"
+          -out "{{ server_cer }}" 
       become: true
 
-    - name: Generate a Self Signed OpenSSL telemetry dsmsroot certificate
-      openssl_certificate:
-        path: "{{ dsmsroot_cer }}"
-        privatekey_path: "{{ dsmsroot_key }}"
-        csr_path: "{{ dsmsroot_csr }}"
-        subject:
-          commonName: ndastreamingclienttest
-        provider: selfsigned
+    - name: Generate dsmsroot cert using openssl. This need to be copied and renamed as streamingtelemetryclient.cer on ptfdocker
```

Comment:
you should add "dsmsroot_key needs to be copied to renamed as streamingtelemetryclient.key on ptfdocker"

Reply:
added a comment to both. Let me know if it's not clear. Thanks